common/src/main/java/io/github/consistencyplus/consistency_plus/core/entries/block/AbstractRegistryEntry.java

```diff
@@ -0,0 +1,74 @@
+package io.github.consistencyplus.consistency_plus.core.entries.block;
+
+
+import dev.architectury.registry.registries.RegistrySupplier;
+import io.github.consistencyplus.consistency_plus.base.ConsistencyPlusMain;
+import io.github.consistencyplus.consistency_plus.blocks.BlockTypes;
+import io.github.consistencyplus.consistency_plus.blocks.BlockShapes;
+import io.github.consistencyplus.consistency_plus.core.entries.interfaces.BlockRegistryInterface;
+import io.github.consistencyplus.consistency_plus.core.extensions.CPlusFenceGateBlock;
+import io.github.consistencyplus.consistency_plus.core.extensions.CPlusStairBlock;
+import io.github.consistencyplus.consistency_plus.registry.CPlusEnhancedRegistry;
+import net.minecraft.block.*;
+import net.minecraft.item.Item;
+
+import static io.github.consistencyplus.consistency_plus.registry.CPlusEnhancedRegistry.checkMinecraft;
+
+public abstract class AbstractRegistryEntry implements BlockRegistryInterface {
```

Comment:
I don't understand this name. A registry entry is a Block or an Item, this is like an entry group.

Reply:
Idk I couldn't think of a good name.

Reply:
RegistryEntryGroup?